cassiopeia/preprocess/pipeline.py

```diff
@@ -577,22 +624,81 @@ def call_alleles(
     return alignments
 
 
+def error_correct_intbcs(
```

Comment:
I would prefer to call this something else because we already have an `error_correct_intbc` step that can be done on molecule tables, correcting intBCs to one another if they share the same allele and cellBC. (This is in `cassiopeia/preprocess/utilities.py` and invoked in `pipeline.filter_molecule_table`)

Why don't we call this `error_correct_intbc_to_whitelist`? 

The naming convention in the setup script and configuration files should also be amended to reflect whatever name you choose.